src/app/plugins/pubsub.plugin.ts

```diff
@@ -79,8 +82,9 @@ export class PubSubPlugin extends Plugin {
     return this._EMBED_LIST;
   }
 
-  private _generateEmbedSub(subData: subData): MessageEmbed {
+  private _generateEmbedSub(subData: subData, guild: Guild | null): MessageEmbed {
```

Comment:
Maybe<Guild>

Reply:
You dont need to pass in guild, and avoid the maybe altogether is you just use 
`this.container.guildSerivce.get()`

Reply:
using "this.container.guildService.get()" defeats the purpose of me moving it into the function

Reply:
how?

Reply:
Because I moved it into the function so it is multi-server friendly, and as far as I know the only way to track what server a message comes from is via referring to the message itself.